diff -Naur taskflow-0.2.orig/taskflow/version.py taskflow-0.2/taskflow/version.py
--- taskflow-0.2.orig/taskflow/version.py	2014-04-01 22:21:14.000000000 +0000
+++ taskflow-0.2/taskflow/version.py	2014-05-07 14:52:41.940019661 +0000
@@ -14,21 +14,32 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
-from pbr import version as pbr_version
+class VersionInfo(object):
+    version = "REDHATVERSION"
+    release = "REDHATRELEASE"
 
-TASK_VENDOR = "OpenStack Foundation"
-TASK_PRODUCT = "OpenStack TaskFlow"
-TASK_PACKAGE = None  # OS distro package version suffix
+    def version_string(self):
+        return self.version
 
-version_info = pbr_version.VersionInfo('taskflow')
+    def cached_version_string(self):
+        return self.version
+
+    def release_string(self):
+        return self.release
+
+    def canonical_version_string(self):
+        return self.version
+
+    def version_string_with_vcs(self):
+        return self.release
+
+
+version_info = VersionInfo()
 
 
 def version_string():
-    return version_info.version_string()
+    return version_info.version
 
 
 def version_string_with_package():
-    if TASK_PACKAGE is None:
-        return version_string()
-    else:
-        return "%s-%s" % (version_string(), TASK_PACKAGE)
+    return "%s-%s" % (version_info.version, version_info.release)
